recipes/conmon/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set version = "2.0.21" %}
+
+package:
+  name: conmon
+  version: {{ version }}
+
+source:
+  url: https://github.com/containers/conmon/archive/v{{ version }}.tar.gz
+  sha256: 03c357c2ee35317e781111d59c4f3fb34033e77a17a8f4221f2ed6d3bcc10c25
+  patches:
+    - 0001-Add-missing-defines-for-CentOS-6.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - glib >=2.0
+    # Header+pkgconfig files for libglib are in the glib package.
+    # Explicitly depend on libglib to avoid glib run_exports from older builds.
+    - libglib >=2.0
```

This prevents the global pinning from being applied:
```suggestion
    - glib
    # Header+pkgconfig files for libglib are in the glib package.
    # Explicitly depend on libglib to avoid glib run_exports from older builds.
    - libglib
```
I'm not sure `libglib` is needed in view of the global pin being used in the feedstock?

---

Looks like our pin is for `2.58` (https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/9139cdcd01e237239f4bbc6241e0bf7cf19b2923/recipe/conda_build_config.yaml#L425) but the split is only available for version `>=2.66.1` (https://github.com/conda-forge/glib-feedstock/pull/57/files) and `2.64.6` (https://github.com/conda-forge/glib-feedstock/pull/68).
Hmm... I'll definitely have to add a `pin_compatible` though. The `run_exports` uses the default `max_pin` -- that one is just `'x'`, right (I can't remember exactly)? If so, keeping the version constraint and adding a `pin_compatible` should work.

---

Oh wait, duh, if I depend on `libglib`, I always get the `run_exports` and as such don't need a `pin_compatible`. Nvm.